Added an alternative way to enable traefik dashboard (#99)
rancher-sandbox/docs.rancherdesktop.io: docs/faq.md

#### **Q: How can I enable the dashboard for the Traefik ingress controller?**

**A:** The Traefik dashboard is not exposed by default, for security reasons. However, it is possible to expose the dashboard in multiple ways. For instance, You can use one of the two approaches shown below.

> #### Using *port-forward* to enable dashboard access

Reviewer: ```suggestion
#### Using `port-forward` to enable dashboard access
```

```
kubectl port-forward -n kube-system $(kubectl -n kube-system get pods --selector "app.kubernetes.io/name=traefik" --output=name) 9000:9000
```

Visit [http://127.0.0.1:9000/dashboard/](http://127.0.0.1:9000/dashboard/) in your browser to view the Traefik dashboard.

> #### Using *HelmChartConfig* to enable dashboard access

Reviewer: ```suggestion
#### Using `HelmChartConfig` to enable dashboard access
```

> Copy below instructions into a file, for example, *ExposeTraefik.yaml*

Reviewer: ```suggestion
Copy the instructions below into a file, for example, `expose-traefik.yaml`
```

Reviewer: I think snake or kebab case may be more common than camel case after looking at some examples in the k8s repo.

E.g. https://github.com/kubernetes/kubernetes/tree/master/cmd/kubeadm/app/util/config/strict/testdata and https://github.com/kubernetes/kubernetes/tree/master/test/integration/scheduler_perf/config
